.github/workflows/ci.yaml

```diff
@@ -273,9 +273,7 @@ jobs:
       - determine_changes
     # Only runs on pull requests, since that is the only we way we can find the base version for comparison.
     # Ecosystem check needs linter and/or formatter changes.
-    if: github.event_name == 'pull_request' && ${{
-      needs.determine_changes.outputs.code == 'true'
-      }}
```

Comment:
The syntax is incorrect here; this currently seems to always evaluate to `true` when I sync the `main` branch of my fork with the upstream `main`